content/100-getting-started/index.mdx

```diff
@@ -37,6 +38,7 @@ Explore our products that make it easy to build and scale data-driven applicatio
 
 [**<SignalStream />Prisma Pulse**](/pulse) allows you to react to database changes with type-safe model streams in real-time.
 
+[**<Inspect />Prisma Optimize**](/optimize/) helps you analyze queries, generate insights, and provides recommendations to make your database queries faster.
```

Comment:
analyze queries and generate insights is the same thing no?

Reply:
@mhessdev Analyzing queries focuses on how well or efficiently the queries operate, whereas generating insights focuses on what the data tells us after being processed by those queries. These are related but distinct activities, so they are not the same thing.